### PR TITLE
Add Verdaccio chart

### PR DESCRIPTION
**Pull request by etiennetremel**
This PR add a new stable chart for [Verdaccio](http://www.verdaccio.org/) (Lightweight private npm proxy registry).

### CODE REVIEW

**Comment by etiennetremel** [comment]
CLA signed

File: stable/verdaccio/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: A lightweight private npm proxy registry (sinopia fork)
+name: verdaccio
+version: 0.1.0
```
Comment (unguiculus) [inline stable/verdaccio/Chart.yaml:4]: Add `appVersion`

File: stable/verdaccio/README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+## Introduction
+
+This chart bootstraps a [Verdaccio](https://github.com/verdaccio/verdaccio)
+deployment on a [Kubernetes](http://kubernetes.io) cluster using the
+[Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
```
Comment (unguiculus) [inline stable/verdaccio/README.md]: We support the current and the previous minor version. So, 1.7+ would be correct.

File: stable/verdaccio/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+apiVersion: extensions/v1beta1
```
Comment (unguiculus) [inline stable/verdaccio/templates/deployment.yaml]: `apiVersion: apps/v1beta1`

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**